R/zzz.R

```diff
@@ -172,9 +174,7 @@ all_undesirable_functions <- modify_defaults(
   untrace = paste(
     "remove this likely leftover from debugging.",
     "It is only useful for interactive debugging with trace()"
-  ),
-  structure =
```

Comment:
The rest of the object is sorted, I think we tacked this on the end by mistake. Now it's sorted again.

(we can't really regression test this because `modify_defaults()` will return it sorted -- the benefit is only for ease of reading the source code)